migration/sql-files/107-remove-link-category-concept.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE work_item_link_types DROP COLUMN link_category_id;
+DROP TABLE work_item_link_categories;
```

Comment:
Hum, I'm a bit skeptical here: is there any risk of SQL error during the rollout of the new version on production ? My point is, I would rather remove the table in a subsequent deployment.

Reply:
@xcoulon I don't get your point. I'm dropping the column on the link type first to avoid that an automatic cascaded delete kicks in and deletes the types connected to a category. Then when there's no one referencing the category table, I remove it. The order is perfectly fine this way.

Reply:
no, my point is: with multiple instances on production, the first upgrade will trigger the database migration, while the older replicas might still use it, because they are still running an older version of the code.

Reply:
Thank you @xcoulon for pointing this out. I've addressed this here: 7a22712f17ffa6ec2fbc208a01d1a5a2e1e66a93.